Kernel/Storage/SATADiskDevice.cpp

```diff
@@ -60,7 +60,7 @@ void SATADiskDevice::start_request(AsyncBlockDeviceRequest& request)
 
 String SATADiskDevice::device_name() const
 {
-    return String::formatted("hd{:c}", 'a' + minor());
+    return String::formatted("sd{:c}", 'a' + minor());
```

Comment:
This is not a typo, but a name that was decided because it represents "hard drive" for the letters `hd`.
As far as I know, `sd` represents a compatibility with the `SCSI`  subsystem in Linux (which means that almost everything besides NVMe emulates SCSI).

Reply:
What happens if a system has both IDE and AHCI drives at the same time?

Reply:
Ideally, the kernel shouldn't worry about naming of storage devices, so we should have a userspace `DeviceMapper` (as a service in `SystemServer`) for such purpose (look at #5757 for this).
for the time being, we just call a method from the `StorageManagement` singleton to handle the decision of the "minor number", so there's no way that both drives will have the same name.